saleor/demo/management/commands/populatedb.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+
+from ....core.management.commands.populatedb import Command as PopulateDBCommand
+from ....extensions.manager import get_extensions_manager
+from ....payment.gateways.braintree.plugin import BraintreeGatewayPlugin
+
+
+def configure_braintree():
+    braintree_api_key = getattr(settings, "BRAINTREE_API_KEY", "")
+    braintree_merchant_id = getattr(settings, "BRAINTREE_MERCHANT_ID", "")
+    braintree_secret = getattr(settings, "BRAINTREE_SECRET_API_KEY", "")
+
+    if not (braintree_api_key and braintree_merchant_id and braintree_secret):
+        return False
+
+    manager = get_extensions_manager()
+    manager.get_plugin_configuration(BraintreeGatewayPlugin.PLUGIN_NAME)
+    manager.save_plugin_configuration(
+        BraintreeGatewayPlugin.PLUGIN_NAME,
+        {
+            "active": True,
+            "configuration": [
+                {"name": "Public API key", "value": braintree_api_key},
+                {"name": "Merchant ID", "value": braintree_merchant_id},
+                {"name": "Secret API key", "value": braintree_secret},
+                {"name": "Use sandbox", "value": True},
+            ],
+        },
+    )
+    return True
+
+
+class Command(PopulateDBCommand):
+    def handle(self, *args, **options):
+        super().handle(*args, **options)
+        is_configured = configure_braintree()
+        if is_configured:
+            self.stdout.write("Configured Braintree")
+        else:
+            self.stdout.write(
+                "Failed to configure Braintree. Check if proper environment variables "
```

Comment:
This is only a suggestion but for someone who doesn't have any knowledge about populateDB or braintree, this error can be hard to understand. What do you think about adding some str to this msg like: `Check if Braintree variables: BRAINTREE_API_KEY, BRAINTREE_API_KEY, BRAINTREE_API_KEY are set`
Also, I wouldn't use the word `Failed` as this can trigger a lot of issues from the community. I would use something like - `Braintree env variables(BRAINTREE_API_KEY, BRAINTREE_API_KEY, BRAINTREE_API_KEY) not found, using DUMMY gateway as a defualt`

Reply:
The community is not supposed to use this demo configuration. But I can names these vars explicitly if you like.

Reply:
Ach, my bad. I thought that you changed  the standard `populatedb` command. Please, ignore this comment

Reply:
Thank you, sir.